features/ticketing/jira-client/dependency-reduced-pom.xml

```diff
@@ -1,77 +1,78 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This seems out of place - this file should probably be deleted and added to .gitignore instead.

Reply:
Yeah, that file snuck into the commit after I did a build. I'll clean it up.